code/drasil-website/Website/Drasil_Website.html

```diff
@@ -0,0 +1,426 @@
+              <td>NoL</td>
+              <td>U</td>
+              <td>WI</td>
+              <td>V</td>
+              <td>D</td>
+            </tr>
+            <tr>
+              <td>Projectile_U_P_L_B_WI_V_F</td>
+              <td>U</td>
+              <td>P</td>
+              <td>L</td>
+              <td>B</td>
+              <td>WI</td>
+              <td>V</td>
+              <td>F</td>
+            </tr>
+          </table>
+        </div>
+        <ul class="list">
+          <li>
+            Modularity
+            <ul class="list">
+              <li>U - Unmodular</li>
+              <li>C - Modular with Combined input module</li>
+              <li>S - Modular with Separated input module</li>
+            </ul>
+          </li>
+          <li>
+            Implementation Type
+            <ul class="list">
+              <li>P - Program</li>
+              <li>L - Library</li>
+            </ul>
+          </li>
+          <li>
+            Logging
+            <ul class="list">
+              <li>NoL - No Logging statements</li>
+              <li>L - Logging statements included</li>
+            </ul>
+          </li>
+          <li>
+            Input Structure
+            <ul class="list">
+              <li>B - Inputs are Bundled in a class</li>
+              <li>U - Inputs are Unbundled</li>
+            </ul>
+          </li>
+          <li>
+            Constant Structure
+            <ul class="list">
+              <li>I - Constant values are Inlined</li>
+              <li>WI - Constants are stored With the Inputs</li>
+              <li>
+                B - Constants are stored in variables that are Bundled in a class
+              </li>
+              <li>U - Constants are stored in variables that are Unbundled</li>
+            </ul>
+          </li>
+          <li>
+            Constant Representation
+            <ul class="list">
+              <li>V - Constants are stored as Variables</li>
+              <li>C - Constants are stored as Constants</li>
+            </ul>
+          </li>
+          <li>
+            Real Number Representation
+            <ul class="list">
+              <li>D - Real numbers are represented as Doubles</li>
+              <li>F - Real numbers are represented as Floats</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <div id="Sec:Examples">
+      <div class="section">
+        <h1>Generated Examples</h1>
+        <p class="paragraph">
+          Each of the case studies contain their own generated PDF and HTML reports, and in some cases, their own generated code.
+        </p>
+        <ul class="list">
+          <li>
+            DblPendulum 
+            <ul class="list">
+              <li>
+                DblPendulumSRS <a href="/home/ant13731/Drasil/code/deploy/examples/DblPendulum/srs/DblPendulum_SRS.html">[HTML]</a> <a href="/home/ant13731/Drasil/code/deploy/examples/DblPendulum/srs/DblPendulum_SRS.pdf">[PDF]</a>
```

Comment:
Links will break here. I think you can just use relative hrefs instead of absolute hrefs.